src/styles/sidebar/components/_index.scss

```diff
@@ -37,3 +34,18 @@
 @use './ToastMessages';
 @use './TopBar';
 @use './VersionInfo';
+
+// TODO: Put these in @layer components after shared component styles have
+// been converted to Tailwind (and are contained in the components layer
+// themselves). They need to come after shared-component styles in the
+// output stylesheet.
```

Comment:
This is a bit of a bummer, but at least I believe it to be temporary.

Styles are output in the following order:

* Resets and anything else in the TW `base` layer
* Things in the TW `components` layer
* Things in the TW `utilities` layer
* Things that are not in a TW layer

Neither shared-component styles nor remaining non-Tailwind-styled local component styles are in a layer (YET). The desire here is to have these "pattern" styles at the very end of the `components` layer such that they override shared and other local component styles. They need to come after the other component-related styles. But if I put them in a components layer _now_, they'll end up getting output _before_ the current component styles, which aren't in a layer at all. I hope that makes any kind of sense.

Reply:
This discussion about layers reminded me of a question I had about TW and CSS Cascade Layers. Is the plan that TW will ultimately map its layers to CSS layers, when browser support is ubiquitous enough?

Reply:
> This discussion about layers reminded me of a question I had about TW and CSS Cascade Layers. Is the plan that TW will ultimately map its layers to CSS layers, when browser support is ubiquitous enough?

I can only imagine _yes_, though curiously, it doesn't seem like anyone is talking about it (i.e. I can't google it up). The syntax used is the same as the spec, minus the definition of what order the layers should be in. I can't imagine that TW wouldn't support native layers when it gets more widespread, but I do wish I could find some mention of it somewhere.